src/Vfs.cc

```diff
@@ -0,0 +1,30 @@
+// Copyright © 2023. GothicKit Contributors
+// SPDX-License-Identifier: MIT
+#include "Prelude.h"
+
+#include <phoenix/Vfs.hh>
+#include <phoenix/cffi/Vfs.h>
+
+PxVfs* pxVfsNew() {
+	return RC(PxVfs, new phoenix::Vfs());
```

Comment:
This `RC(...)` wrapper is not required since `PxVfs` is already an alias for `phoenix::Vfs`.